Update career info (#49)
facile-it/facile-it.github.io: content/ita/careers.md

![Facile.it - lavora con noi!](/images/careers/banner.png)

> **Facile.it** is constantly growing and we are always looking for programmers to join our team.

Author: I'm agree with you, maybe we can use a more generic name for all IT roles.
How about replacing `we are always looking for programmers` with "we are always looking for experts to join in our IT Area"? 
Do you have any suggestions?

> **Facile.it** is constantly growing and we are always looking for programmers to join our team.

Reviewer: I'm not sure if the HR department would agree on the term "expert" since we also hire juniors. If we want to use the term expert we should probably check with them
What about `we are always looking for IT professionals to join our team`?

> This proposal is aimed at highly experienced developers who believe in innovation and are always attentive to new technologies and can't wait to put them into practice. We are a dynamic, young and fun team.

Reviewer: This proposal is aimed at experienced developers who believe in innovation and are always on the lookout for new technologies and can't wait to put them to work.

> We are more than 100 programmers and manage numerous projects with different technology stacks.

Reviewer: I would choose either developers or programmers and use the same term in the entire document.
Again, are we sure we want to talk about programmers/developers and not about other figures?

Author: Ditto
We can define them with generic 'experts' or we can list some of the roles, Do you have any suggestions?

Reviewer: See comment above: we could choose the term "professionals".

> Siamo in totale quasi 100 programmatori e gestiamo numerosi progetti con differenti stack tecnologici.

Reviewer 2: Ormai siamo + di 100!!!

Author: Riporto ad HR :) Grazie!

# What we offer at Facile.it:

> * Permanent employment with remuneration commensurate with experience and supplementary health fund

Reviewer 3: ```suggestion
* Permanent employment with remuneration commensurate to your experience level, and supplementary health fund
```

Author: Thanks 🙏🏻

> * Permanent employment with remuneration commensurate to your experience level, and supplementary health fund

Reviewer: I would add a full stop at the end of each sentence.

> * A young and dynamic IT team with an open mind to new technologies and processes

Reviewer: A young, dynamic IT team that is open to new technologies and processes.

> * Personalized induction and growth plans

Reviewer 3: ```suggestion
* Personalized introduction and growth plans
```
Also this may need a reword, we just changed how we handle this..

> * Personalized introduction and growth plans

Reviewer: Personalized onboarding and career plan.

> * 1 Friday per month dedicated to sharing and studying with the team

Reviewer 3: ```suggestion
* 1 Friday per month dedicated to learning, sharing and studying with the whole IT department
```

> * 1 Friday per month dedicated to learning, sharing and studying with the whole IT department

Reviewer: One Friday a month dedicated to learning, sharing and studying with colleagues in the IT department.

> * Personal study budget

Reviewer: Personal budget for training and professional learning.

> * Smart Working 4 days a week and the possibility of working in Full Remote Working

Reviewer 3: ```suggestion
* Smart Working 4 days a week or the possibility of working full remote
```

> * Smart Working 4 giorni a settimana

Reviewer 4: Secondo te, ha senso menzionare la possibilità di essere full remote?

> * Smart Working 4 days a week or the possibility of working full remote

Reviewer: "Smart Working" meaning "work from home" is an incorrect use of the term in Italian.
I would write as follows:
Remote work 4 days per week or 100% of the time (based on agreement).

* Biblioteca di libri di tecnologia con più di 100 titoli   
* Partecipazione ad eventi di tecnologia

> * Ufficio in centro a Milano (MM3 Lodi T.I.B.B.) con sala relax con sofà, ping-pong e calcetto

Reviewer 3: In realtà per ora non ci sono...
Poi ha senso citare Cagliari?

> * Office in the center of Milan (MM3 Lodi T.I.B.B.) with relaxation room and five-a-side football

Reviewer: Office in the center of Milan (MM3 Lodi T.I.B.B.) with relax room and foosball table.

> * Conventions (bars, restaurants, gyms, etc.), ticket Restaurant, coffee and water

Reviewer: Conventions (with bars, restaurants, gyms, etc.), meal vouchers, free coffee and water.

* The opportunity to do the job you love in a successful Italian company with a team of excellence by your side

> If you are interested, take a look at ours {{< url-link "Job opportunities »" "https://facile.jobs.personio.de/#department-538752" "_blank">}}.

Reviewer: (...) take a look at our job openings
